api/v1beta1/kafkacluster_types_test.go

```diff
@@ -425,3 +426,34 @@ func TestGetBrokerConfigEnvs(t *testing.T) {
 		t.Error("Expected:", expected, "Got:", result)
 	}
 }
+
+func TestGetBrokerLabels(t *testing.T) {
```

Comment:
Can you add a comment to the unit test explaining that there are some reserved labels that cannot be overridden?

Reply:
@amberimam The reserved labels are listed here in the CRD: https://github.com/banzaicloud/koperator/pull/803/files#diff-73f611069524d70545c39bf39b1c595ebd06404aae90f7b3be02775d7de77671R174

Not sure if it is a good idea to copy that info over to the simple unit test